apps/wallet/src/features/Swap/Success.tsx

```diff
@@ -0,0 +1,49 @@
+import type { ViewStyle } from 'react-native';
+import { StyleSheet } from 'react-native';
+import { Text, View } from '@walless/gui';
+import { Switch } from '@walless/icons';
+import { useSafeAreaInsets, useSnapshot } from 'utils/hooks';
+
+import { swapContext } from './context';
+
+const Success = () => {
+	const insets = useSafeAreaInsets();
+	const containerStyle: ViewStyle = {
+		...styles.container,
+		paddingTop: Math.max(insets.top, 20),
+		paddingBottom: 20,
+	};
+
+	const { fromToken, amount, toToken, swapQuote } =
+		useSnapshot(swapContext).swap;
+
+	const fromSymbol = fromToken?.metadata?.symbol;
+	const toSymbol = toToken?.symbol;
+	const outAmount = swapQuote
+		? (parseInt(swapQuote.outAmount) * 1.0) / 10 ** (toToken?.decimals || 0)
+		: 0;
```

Comment:
this kind of logic start to repeats, consider an util method for this kind of logic.